bucket/renamer.json

```diff
@@ -5,15 +5,21 @@
     "license": "CC-BY-NC-ND-3.0|Shareware",
     "url": "https://www.den4b.com/download/renamer/portable/7.4?token=scoop#/dl.zip",
     "hash": "1a47a345773ef44222efbe1c1f4c58312470ebd5965fc60d0555d932d620acc2",
-    "pre_install": "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item \"$dir\\Settings.ini\" | Out-Null }",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item \"$dir\\Settings.ini\" -ItemType File | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\Presets\")) { New-Item \"$dir\\Presets\" -ItemType Directory | Out-Null }"
```

Comment:
You don't have to pre-create the directory to persist.

Reply:
Oh your right, I'll fix that.